src/types/AceValidations.ts

```diff
@@ -104,8 +105,8 @@ const parseCheck = (yaml: string, check: ObjectCheck): AceCheck => {
       - spec/destination
       - spec/name (this is used for destination Rules which maps spec/name into destinationName)
       - spec/precedence/<value>
-      - spec/route/weight/<value>
-      - spec/route[<value>]/labels
+      - spec/route[<nWeight>/weight/<value>
```

Comment:
I guess there is missing a `]` right here, isn't it?

Reply:
+1 yeah, it was missing as it is in the comment. Thx.